pandas/_libs/groupby.pyx

```diff
@@ -1806,6 +1896,32 @@ def group_max(
     )
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def group_idxmax(
```

Comment:
NBD but ive been leaning towards avoiding having the extra functions here and instead having `partial`s in WrappedCythonOp._CYTHON_FUNCTIONS.  Makes these files marginally smaller